README.md

```diff
@@ -1,8 +1,89 @@
-ANUGA Github Pages (root)
--------------------------
+# ANUGA Community TechLauncher Project
 
-This could evolve into an official go to for all ANUGA documentation.
+## Overview
 
+The existing ANUGA Hydro software has been used to model floods for more than a decade, with excellent results. But for smaller floods, it has a tendency to over estimate the flood area, due to being unable to model underground drainage systems.
 
+This project will extend the ANUGA Hydro software, which is capable of hydrodynamic modelling, by coupling with the US EPA's Storm Water Management Model (SWMM), thus adding to it the ability to model the effects of underground drainage systems. 
 
+## Team
 
+|  Name          | UID    | Principal Role | Secondary Role | Other |
+|:--------------:|:------:|:--------------:|:--------------:|:-----:|
+| Zhixian (Rachel) Wu | u5807060 | TBA | TBA | Spokesperson, Team Leader |
+| Xingnan Pan | u6744662 | TBA | TBA | Deputy Spokesperson, Project Manager |
+| Chen Huang | u6735118 | TBA | TBA |  |
+| Lixin Hou | u6456457 | TBA | TBA |  |
+| Mingda Zheng | u6686733 | TBA | TBA |  |
+| Yijie Liu | u6890141 | TBA | TBA |  |
+| Zijun Zhang | u6904534 | TBA | TBA |  |
+
+## Stakeholders
+* **The sponsors:**
+   * Professor Stephen Roberts, ANU
+   * Dr Ole Nielsen, Geoscience Australia
+* **The user representatives (flood modellers):**
+   * Rudy Van Drie, Balance Research and Development
+   * Dr Petar Milevski, Civil Engineer Urban Drainage, Wollongong City Council
+
+## Documentation
+
+### Sprint Stories
+
+> [Trello](https://trello.com/b/Z45C7crP/agile-sprint-board)
+
+### Communication
+
+> [Slack](https://anu-flood-modelling.slack.com)
+
+### Meeting Minutes
+
+Sprint 1 (start of semester - 19/08/2020)
+
+> [2020-08-04 Team Meeting](https://docs.google.com/document/d/1SW3PUsRs-9bc1CYlVkW6fHQLiOQ0cm0w_jzSKu37Gpw/edit?usp=sharing)
+> [2020-08-06 Client Meeting](https://docs.google.com/document/d/1J_kqxAhOHSAh3xWV8enVu0XkZSba1jQchf01azwkgvg/edit?usp=sharing)
+
+### Decisions
+
+> [Log for Small Decisions](https://docs.google.com/spreadsheets/d/1uPZlRMNaRBlZnUdfNPVQ4e_S48npiRRkqP9GHJUyXS4/edit?usp=sharing)
+> [Documents for Large Decisions](https://docs.google.com/spreadsheets/d/1uPZlRMNaRBlZnUdfNPVQ4e_S48npiRRkqP9GHJUyXS4/edit?usp=sharing)
+
+## Timeline
+
+We are doing two-week sprints, with client meetings to close each sprint on Wednesday 5:00PM Canberra time, and team meetings for sprint retrospectives and sprint planning on Wednesday 7:00PM Canberra time.
+
+The first sprint will be a bit longer, so that the rest of the sprints will end just before the Week 6 and Week 10 audits. This means the first sprint will end Wednesday of Week 4.
+
+* **2020-s2-timeline**
+<img src="https://drive.google.com/file/d/1fBOS3L8SHISkuszNiWiaJWVJX4TLfvgx/view?usp=sharing" alt="20-s2-TL" align=center />
```

Comment:
The link cannot be visualised in the file. We require to upload the image onto the repo and link it.